internal/cli/root.go

```diff
@@ -93,6 +95,15 @@ func (c *Client) apiConnect(httpClient *http.Client) error {
 	return nil
 }
 
+func (c *Client) metalApiConnect(httpClient *http.Client) error {
+	configuration := metal.NewConfiguration()
+	configuration.Debug = false
```

Comment:
This should be set to true if the `PACKNGO_DEBUG` env var is provided.  Here's an example of how we could do that from the clusterapi provider: https://github.com/kubernetes-sigs/cluster-api-provider-packet/pull/527/files#diff-febb66857beae6d70b0aad1c73c92369afe1fb7d309f699f6047af7912d99e05R369-R374

Reply:
updated